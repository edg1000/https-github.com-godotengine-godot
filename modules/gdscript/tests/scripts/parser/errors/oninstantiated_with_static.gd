extends Node

@oninstantiated static var node

func test():
	pass
