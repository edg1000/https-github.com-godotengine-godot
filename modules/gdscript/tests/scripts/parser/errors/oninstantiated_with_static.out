GDTEST_ANALYZER_ERROR
"@oninstantiated" annotation cannot be applied to a static variable.
